ArkhamOverlay/Services/CardLoadService.cs

```diff
@@ -25,14 +28,63 @@ public class CardLoadService {
             var worker = new BackgroundWorker();
             worker.DoWork += (x, y) => {
                 _loadingStatusService.ReportEncounterCardsStatus(Status.LoadingCards);
+                _logger.LogMessage("Loading encounter cards.");
                 try {
-                    _arkhamDbService.LoadEncounterCards(_appData);
+                    _appData.Game.ScenarioCards.Loading = true;
+                    _appData.Game.LocationCards.Loading = true;
+                    _appData.Game.EncounterDeckCards.Loading = true;
+
+                    var cards = _arkhamDbService.LoadEncounterCards();
+                    cards.AddRange(_localCardsService.LoadEncounterCards());
+
+                    var scenarioCards = new List<Card>();
+                    var agendas = new List<Card>();
+                    var acts = new List<Card>();
+                    var locations = new List<Card>();
+                    var treacheries = new List<Card>();
+                    var enemies = new List<Card>();
+
+                    foreach (var card in cards) {
+                        switch (card.Type) {
+                            case CardType.Scenario:
+                                scenarioCards.Add(card);
+                                break;
+                            case CardType.Agenda:
+                                agendas.Add(card);
+                                break;
+                            case CardType.Act:
+                                acts.Add(card);
+                                break;
+                            case CardType.Location:
+                                locations.Add(card);
+                                break;
+                            case CardType.Treachery:
+                            case CardType.Enemy:
+                                treacheries.Add(card);
+                                break;
+                            default:
+                                break;
+                        }
+                    }
+
+                    scenarioCards.AddRange(agendas);
```

Comment:
Minor: Is there a reason we add these later instead of bundling together the cases above? I assume for the ordering, but just making sure.